### ISSUE TITLE
Short Form remove the first numerical value of the IRI fragment

### ISSUE THREAD

**Issue by Ayllonbe** [EBISPOT/OLS#541]
I do not know if I can modify from my side but I was looking for the documentation and I did not see anything. I have a local OLS (for testing) and I introduce my ontology correctly. However, my ontology has identifier like **TOTO:1TOEP**. The issue that I have with OLS is to how force him to keep the full fragment value in the short form. That means that I expect to have 1TOEP in the short form instead of TOEP. 

Thank you

**Comment by serjoshua** [comment]
hi @Ayllonbe may I know what parts of OLS you experience this? or maybe a screenshot of the issue on your end.

**Comment by henrietteharmse** [comment]
Can you please provide a minimal complete example ontology that illustrates the problem you are experiencing?

**Comment by Ayllonbe** [comment]
Yes, sorry to answer that late. The main issue is found in the _short_term_ tag. This tag is generated based on the IRI. I did explore a bit the code and I do not know if the issue is from your side or rather from OWL API. If it is the later, it is not a real issue from them since they are following the XML convention [QName](https://en.wikipedia.org/wiki/QName).

In my case I have an IRI as follows: https://ontology.toto.net/ontology/MyOntology/1TOEP, but then the short_form only representd TOEP instead 1TOEP. That behaviour is shown in all the API calls results but also in the browser, if I tap 1TOEP I have not autocompletation results but removing the **1** I have the TOEP.

**Comment by henrietteharmse** [comment]
@Ayllonbe  Can you please provide a minimal complete example ontology that illustrates the problem you are experiencing?

**Comment by Ayllonbe** [comment]
You can find attached the ontology (an example one). When I load it into my OLS (Local one and unfourtunately I cannot provide screenshot since my computer is working very weird and give me black screenshot), then the tag short_form in the JSON file is TEST instead of 2TEST

```
<?xml version="1.0"?>
<rdf:RDF xmlns="http://www.semanticweb.org/ayllona/ontologies/2022/0/untitled-ontology-21#"
     xml:base="http://www.semanticweb.org/ayllona/ontologies/2022/0/untitled-ontology-21"
     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
     xmlns:owl="http://www.w3.org/2002/07/owl#"
     xmlns:xml="http://www.w3.org/XML/1998/namespace"
     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
    <owl:Ontology rdf:about="http://www.semanticweb.org/ayllona/ontologies/2022/0/untitled-ontology-21"/>
    


    <!-- 
    ///////////////////////////////////////////////////////////////////////////////////////
    //
    // Classes
    //
    ///////////////////////////////////////////////////////////////////////////////////////
     -->

    


    <!-- https://ontology.basf.net/ontology/2022/BASF_TEST/test#3TEST -->

    <owl:Class rdf:about="https://ontology.basf.net/ontology/2022/BASF_TEST/test#3TEST"/>
    


    <!-- https://ontology.basf.net/ontology/BASF_TEST/2TEST2 -->

    <owl:Class rdf:about="https://ontology.basf.net/ontology/BASF_TEST/2TEST2"/>
</rdf:RDF>



<!-- Generated by the OWL API (version 4.2.8.20170104-2310) https://github.com/owlcs/owlapi -->


```
